jobs/nfcens/plots/JEVS_NFCENS_WAVE_GRID2OBS_PLOTS

```diff
@@ -77,12 +75,9 @@ export jlogfile=${DATA}/${MODELNAME}_${RUN}_plots.log
 ####################################
 # Execute the script                
 ####################################
-env
-$HOMEevs/scripts/${COMPONENT}/${STEP}/exevs_${COMPONENT}_${RUN}_${VERIF_CASE}_${STEP}.sh 1>>$pgmout 2>errfile.${pid}
+$HOMEevs/scripts/nfcens/plots/exevs_nfcens_wave_grid2obs_plots.sh
```

Comment:
@lgannoaa , can you update to use $COMPONENT in place of nfcens, $STEP in place of plots? I would also like to see $RUN in place of wave and VERIF_CASE in place of grid2obs.